### DISCUSSION TITLE
Do you have any interst to support Lora for fast adaption to small datasets from big COCO？

### DISCUSSION THREAD

**Discussion by LYMDLUT** [open-mmlab/mmyolo#648] [Ideas]
Lora ，the most adavenced method for fast adapation，with extremely low parameter and training epochs，using the pretrained COCO ckpt，achieving  compareable performance of fintuing or even better.
http://arxiv.org/abs/2106.09685

**Comment by hhaAndroid** [comment]
@LYMDLUT This is an interesting topic. Does it also work in the visual domain and especially in YOLO algorithms?

**Comment by LYMDLUT** [comment]
https://github.com/LYMDLUT/Lora_with_ResBnLn/blob/main/lora1.py
I'll provide a simple example, and maybe we can make some fair comparisons based on it.
    model.requires_grad_(False)
    lora_layers = warp_with_lora(model, 20)
    lora_layers.requires_grad_(True)
